worlds/ladx/ItemIconGuessing.py

```diff
@@ -0,0 +1,210 @@
+    'CHICKEN': 'ROOSTER',
+    'COJIRO': 'ROOSTER', # Ocarina of Time
+    'CUCCO': 'ROOSTER',
+    'FLY': 'ROOSTER',
+    'GRIFFIN': 'ROOSTER',
+    'GRYPHON': 'ROOSTER',
+    'ARMOR': 'BLUE_TUNIC',
+    'DRAGONARMOR': 'BLUE_TUNIC', # Final Fantasy 1
+    'MAIL': 'BLUE_TUNIC',
+    'SUIT': 'BLUE_TUNIC',
+    'TANK': 'HEART_CONTAINER',
+    'FUNGAL': 'TOADSTOOL',
+    'FUNGUS': 'TOADSTOOL',
+    'MUSHROOM': 'TOADSTOOL',
+    'SHROOM': 'TOADSTOOL',
+    'NUT': 'GUARDIAN_ACORN',
+    'SEED': 'GUARDIAN_ACORN',
+    'KEY': 'KEY', # Without this, keys show up as nightmare keys
+    'FOSSIL': 'STONE_BEAK',
+    'RELIC': 'STONE_BEAK',
+    'BOLERO': 'SONG1',
+    'LULLABY': 'SONG1',
+    'MELODY': 'SONG1',
+    'MINUET': 'SONG1',
+    'NOCTURNE': 'SONG1',
+    'PRELUDE': 'SONG1',
+    'REQUIEM': 'SONG1',
+    'SERENADE': 'SONG1',
+    'SONG': 'SONG1',
+    'FISH': 'SONG2',
+    'SURF': 'SONG2',
+    'FROG': 'SONG3',
+    'CELLO': 'INSTRUMENT1',
+    'GUITAR': 'INSTRUMENT1',
+    'LUTE': 'INSTRUMENT1',
+    'VIOLIN': 'INSTRUMENT1',
+    'HORN': 'INSTRUMENT2',
+    'BELL': 'INSTRUMENT3',
+    'CHIME': 'INSTRUMENT3',
+    'HARP': 'INSTRUMENT4',
+    'MARIMBA': 'INSTRUMENT5',
+    'XYLOPHONE': 'INSTRUMENT5',
+    'TRIANGLE': 'INSTRUMENT6',
+    'KEYBOARD': 'INSTRUMENT7',
+    'ORGAN': 'INSTRUMENT7',
+    'PIANO': 'INSTRUMENT7',
+    'DRUM': 'INSTRUMENT8',
+    'DINOSAUR': 'TRADING_ITEM_YOSHI_DOLL',
+    'DRAGON': 'TRADING_ITEM_YOSHI_DOLL',
+    'TOY': 'TRADING_ITEM_YOSHI_DOLL',
+    'HAIRBAND': 'TRADING_ITEM_RIBBON',
+    'HAIRPIN': 'TRADING_ITEM_RIBBON',
+    'CAN': 'TRADING_ITEM_DOG_FOOD',
+    'BANANA': 'TRADING_ITEM_BANANAS',
+    'BRANCH': 'TRADING_ITEM_STICK',
+    'TWIG': 'TRADING_ITEM_STICK',
+    'BEEHIVE': 'TRADING_ITEM_HONEYCOMB',
+    'HIVE': 'TRADING_ITEM_HONEYCOMB',
+    'HONEY': 'TRADING_ITEM_HONEYCOMB',
+    'FRUIT': 'TRADING_ITEM_PINEAPPLE',
+    'FLOWER': 'TRADING_ITEM_HIBISCUS',
+    'PETAL': 'TRADING_ITEM_HIBISCUS',
+    'CARD': 'TRADING_ITEM_LETTER',
+    'MESSAGE': 'TRADING_ITEM_LETTER',
+    'SWEEP': 'TRADING_ITEM_BROOM',
+    'CLAW': 'TRADING_ITEM_FISHING_HOOK',
+    'AMULET': 'TRADING_ITEM_NECKLACE',
+    'BEADS': 'TRADING_ITEM_NECKLACE',
+    'PEARLS': 'TRADING_ITEM_NECKLACE',
+    'PENDANT': 'TRADING_ITEM_NECKLACE',
+    'ROSARY': 'TRADING_ITEM_NECKLACE',
+    'FINDER': 'TRADING_ITEM_MAGNIFYING_GLASS',
+    'LENS': 'TRADING_ITEM_MAGNIFYING_GLASS',
+    'MIRROR': 'TRADING_ITEM_MAGNIFYING_GLASS',
+    'SCOPE': 'TRADING_ITEM_MAGNIFYING_GLASS',
+    'CHAOS': 'PIECE_OF_POWER', # Chaos emerald
+    'POWER': 'PIECE_OF_POWER',
+    'STAR': 'PIECE_OF_POWER',
+    'TRIFORCE': 'PIECE_OF_POWER',
+
+    #SMZ3
+    'BIGKEY': 'NIGHTMARE_KEY',
+    'SANCTUARYHEARTCONTAINER': 'HEART_CONTAINER',
+    'BOSSHEARTCONTAINER': 'HEART_CONTAINER',
+    'HEARTCONTAINER': 'HEART_CONTAINER',
+    'ENERGYTANK': 'HEART_CONTAINER',
+    'BLUEBOOMERANG': 'BOOMERANG',
+    'MAGICMIRROR': 'TRADING_ITEM_MAGNIFYING_GLASS',
+
+    # TODO: Also use AP item name
```

Comment:
Is this todo still valid?

Reply:
Its a comment left in from the original implementation, I'm not entirely sure what zig meant with it tbh.